src/tests/JIT/HardwareIntrinsics/Arm/Shared/_SveUnaryOpTestTemplate.template

```diff
@@ -75,154 +71,91 @@ namespace JIT.HardwareIntrinsics.Arm
-
-            private static unsafe void* Align(byte* buffer, ulong expectedAlignment)
-            {
-                return (void*)(((ulong)buffer + expectedAlignment - 1) & ~(expectedAlignment - 1));
-            }
-        }
-
         private struct TestStruct
         {
-            public {Op1VectorType}<{Op1BaseType}> _fld1;
+            public Vector<{Op1BaseType}> _fld1;
 
-            public static TestStruct Create()
+            public static TestStruct Create(Vector<{Op1BaseType}> vec)
             {
                 var testStruct = new TestStruct();
-
-                for (var i = 0; i < Op1ElementCount; i++) { _data1[i] = {NextValueOp1}; }
-                Unsafe.CopyBlockUnaligned(ref Unsafe.As<{Op1VectorType}<{Op1BaseType}>, byte>(ref testStruct._fld1), ref Unsafe.As<{Op1BaseType}, byte>(ref _data1[0]), (uint)Unsafe.SizeOf<{Op1VectorType}<{Op1BaseType}>>());
-
+                testStruct._fld1 = vec;
                 return testStruct;
             }
 
             public void RunStructFldScenario({TemplateName}UnaryOpTest__{TestName} testClass)
             {
                 var result = {Isa}.{Method}(_fld1);
 
-                Unsafe.Write(testClass._dataTable.outArrayPtr, result);
-                testClass.ValidateResult(_fld1, testClass._dataTable.outArrayPtr);
+                testClass.ValidateResult(_fld1, result);
             }
         }
 
         private static readonly int LargestVectorSize = {LargestVectorSize};
 
-        private static readonly int Op1ElementCount = Unsafe.SizeOf<{Op1VectorType}<{Op1BaseType}>>() / sizeof({Op1BaseType});
-        private static readonly int RetElementCount = Unsafe.SizeOf<{RetVectorType}<{RetBaseType}>>() / sizeof({RetBaseType});
+        private static readonly int Op1ElementCount = Unsafe.SizeOf<Vector<{Op1BaseType}>>() / sizeof({Op1BaseType});
+        private static readonly int RetElementCount = Unsafe.SizeOf<Vector<{RetBaseType}>>() / sizeof({RetBaseType});
 
-        private static {Op1BaseType}[] _maskData = new {Op1BaseType}[Op1ElementCount];
-        private static {Op1BaseType}[] _data1 = new {Op1BaseType}[Op1ElementCount];
-
-        private {Op1VectorType}<{Op1BaseType}> _mask;
-        private {Op1VectorType}<{Op1BaseType}> _fld1;
-        private {Op1VectorType}<{Op1BaseType}> _falseFld;
-
-        private DataTable _dataTable;
+        private Vector<{Op1BaseType}> _mask;
+        private Vector<{Op1BaseType}> _fld1;
+        private Vector<{Op1BaseType}> _falseFld;
+        private TestLibrary.Vectors.PinnedVector<{Op1BaseType}> _pinnedOp1;
 
         public {TemplateName}UnaryOpTest__{TestName}()
         {
             Succeeded = true;
 
-            for (var i = 0; i < Op1ElementCount; i++) { _maskData[i] = ({Op1BaseType})({NextValueOp1} % 2); }
-            Unsafe.CopyBlockUnaligned(ref Unsafe.As<{Op1VectorType}<{Op1BaseType}>, byte>(ref _mask), ref Unsafe.As<{Op1BaseType}, byte>(ref _maskData[0]), (uint)Unsafe.SizeOf<{Op1VectorType}<{Op1BaseType}>>());
```

Comment:
Removing all the `CopyBlockUnaligned()` and `Unsafe.Write()` calls will hopefully stop all the horrible to debug stress test errors we were seeing earlier in the year.